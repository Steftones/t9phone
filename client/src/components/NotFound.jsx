import React from 'react';

const NotFound = () => (
  <div className="section is-flex is-justify-content-center">
    <h1>
      Oops! Page not found. <a href="/">Go back!</a>
    </h1>
  </div>
);

export default NotFound;
