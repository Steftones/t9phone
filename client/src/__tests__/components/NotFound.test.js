import { render } from "@testing-library/react";
import NotFound from "../../components/NotFound"

describe('NotFound', () => {
  test('it should render correctly', () => {
    const { asFragment } = render(<NotFound/>);
    expect(asFragment()).toMatchSnapshot();
  });
});