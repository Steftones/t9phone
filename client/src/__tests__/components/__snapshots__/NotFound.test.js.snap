// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound it should render correctly 1`] = `
<DocumentFragment>
  <div
    class="section is-flex is-justify-content-center"
  >
    <h1>
      Oops! Page not found. 
      <a
        href="/"
      >
        Go back!
      </a>
    </h1>
  </div>
</DocumentFragment>
`;

exports[`NotFound it should render the component 1`] = `
<DocumentFragment>
  <div
    class="section is-flex is-justify-content-center"
  >
    <h1>
      Oops! Page not found. 
      <a
        href="/"
      >
        Go back!
      </a>
    </h1>
  </div>
</DocumentFragment>
`;
